docs/api/session.md

```diff
@@ -980,12 +980,17 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
          and this is set to `true`, then local playback of audio will not be muted (e.g. using `MediaRecorder`
          to record `WebFrameMain` with this flag set to `true` will allow audio to pass through to the speakers
          while recording). Default is `false`.
+* `opts` Object (optional)
```

Comment:
We can tell the docs parser this is  experimental & macOS:

```suggestion
* `opts` Object (optional) _macOS_ _Experimental_
```

Reply:
Done, thanks!